Homework_1/scripts/demo.jl

```diff
@@ -0,0 +1,148 @@
+# Script to compare the performace of our QR decomposition with the one from LinearAlgebra
+
+using LinearAlgebra
+using Statistics
+using Homework_1
+using Plots
+
+possible_sizes = [10, 50, 100, 250, 500, 750, 1000]
+
+# Create a random symmetric tridiagonal matrix of size n*n
+function create_tridiagonal(n)
+    # Generate random diagonal and sub-diagonal elements
+    diag = randn(n)  # Main diagonal
```

Comment:
Funkcija `create_tridiagonal` ni ponovljiva. Vsakokrat vrne drugačen rezultat. Smiselno je kot argument podati referenco na generator naključnih števil in ga uporabiti v klicih funkcije `randn`.